### PR TITLE
chore: Release notes 2021.02

### LINKED ISSUE #5466: release notes 2021.02
**Describe the issue. What is the expected and unexpected behavior?**
release notes 2021.02

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5466

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,82 @@ id: Release notes
...
+
+### Topology
+- Fixed if statement ([#5409](https://github.com/patternfly/patternfly-react/pull/5409))
+
+### Virtualized extension
+- Revert some changes to VirtualTableBody ([#5434](https://github.com/patternfly/patternfly-react/pull/5434))
+
+### Other
+- **Deps:**
+  - Updated dependency theme-patternfly-org to v0.4.11 ([#5371](https://github.com/patternfly/patternfly-react/pull/5371))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:79]: Don't need this -- superseded by next line.

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,82 @@ id: Release notes
...
+  - Fixed to not rely on toISOString ([#5369](https://github.com/patternfly/patternfly-react/pull/5369))
+- **Drawer:**
+  - Enabled touch controls for resizable drawer ([#5357](https://github.com/patternfly/patternfly-react/pull/5357))
+  - Updated transitions and content mounting ([#5426](https://github.com/patternfly/patternfly-react/pull/5426))
+- **Form select:**
+  - Added support for placeholder styled option ([#5428](https://github.com/patternfly/patternfly-react/pull/5428))
+- **Jumplinks:**
+  - Added jumplinks subsections ([#5314](https://github.com/patternfly/patternfly-react/pull/5314))
+- **Navigation:**
+  - Added support for spreading props to the NaveExpandable button ([#5438](https://github.com/patternfly/patternfly-react/pull/5438))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:39]: NaveExpandable -> NavExpandable

**Review by redallen — CHANGES_REQUESTED** [review]
Just 2 things

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,82 @@ id: Release notes
 section: developer-resources
 ---
 
+## 2021.02 release notes (2021-02-17)
+- [@patternfly/react-catalog-view-extension@4.10.0](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.10.0])
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:7]:
```suggestion
- [@patternfly/react-catalog-view-extension@4.10.0](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.10.0)
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,82 @@ id: Release notes
...
+- **Select:**
+  - Adjusted Select component so that it also works with children changes while it's mounted ([#5375](https://github.com/patternfly/patternfly-react/pull/5375))
+  - Updated to not use focus trap when no tabbable children ([#5427](https://github.com/patternfly/patternfly-react/pull/5427))
+  - Removed onFilter function from example with custom objects ([#5450](https://github.com/patternfly/patternfly-react/pull/5450))
+- **Slider:**
+  - Added touch support ([#5453](https://github.com/patternfly/patternfly-react/pull/5453))
+- **Spinner:**
+  - Added isSVG prop ([#5390](https://github.com/patternfly/patternfly-react/pull/5390))
+- **Time picker:**
+  - Fixed issue when clicking dropdown menu does not trigger onchange function ([#5399](https://github.com/patternfly/patternfly-react/pull/5399))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:55]:
```suggestion
  - Fixed issue when clicking dropdown menu does not trigger onChange function ([#5399](https://github.com/patternfly/patternfly-react/pull/5399))
```

onchange => onChange

**Comment by evwilkin** [comment]
Just one last nit about `onChange` then looks good

**Review by nicolethoen — DISMISSED** [review]

**Review by redallen — APPROVED** [review]

**Review by redallen — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**